src/hooks/useActiveDescendantListFocus/index.ts

```diff
@@ -7,29 +7,39 @@ import {
   useCleanVisualFocusOnBlur
 } from "./useActiveDescendantListFocusHooks";
 
-const ROLES = {
-  APPLICATION: "application",
-  COMBOBOX: "combobox",
-  COMPOSITE: "composite",
-  GROUP: "group",
-  TEXTBOX: "textbox",
-  MENU: "menu"
-};
+enum Roles {
+  APPLICATION = "application",
+  COMBOBOX = "combobox",
+  COMPOSITE = "composite",
+  GROUP = "group",
+  TEXTBOX = "textbox",
+  MENU = "menu"
+}
 
 function useActiveDescendantListFocus({
   focusedElementRef, // the reference for the component that listens to keyboard
   itemsIds,
   isItemSelectable,
   onItemClick,
   defaultVisualFocusFirstIndex = false,
-  focusedElementRole = ROLES.GROUP,
+  focusedElementRole = Roles.GROUP,
   isHorizontalList = false,
   useDocumentEventListeners = false,
   isIgnoreSpaceAsItemSelection = false
+}: {
+  focusedElementRef: MutableRefObject<HTMLElement>;
+  itemsIds: string[];
+  isItemSelectable: (index: number) => boolean;
+  onItemClick: (event: KeyboardEvent | MouseEvent, index: number) => unknown;
```

Comment:
any reason for `unknown` returned value?
if the returned value is never used, a `void` should be good too
```suggestion
  onItemClick: (event: KeyboardEvent | MouseEvent, index: number) => void;
```

Reply:
@laviomri as i don't know how people used this i don't want to break their code, i wanted to use void but we might break someone